grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -54,9 +54,9 @@ interface ClientPageState {
 })
 export class ClientPageStore extends ComponentStore<ClientPageState> {
   constructor(
-      private readonly httpApiService: HttpApiService,
-      private readonly configService: ConfigService,
-      private readonly configFacade: ConfigFacade,
+    private readonly httpApiService: HttpApiService,
```

Comment:
Your PR has hundreds of lines reformatted with whitespace changes. This pollutes the `git blame` view and makes it hard to track actual changes. Also, our internal formatter will override it again.

Please use [clang format](https://github.com/xaverh/vscode-clang-format-provider) to format your code. Use `"clang-format.style": "{BasedOnStyle: google}",` in the settings. If this does not work for you, please try https://github.com/google/gts.